Control server-info location

trlinkin opened the pull request:
This commit enhances the control over the server-info location. It
includes the ability to enable/disable it, as well as controll the
allow list.

trlinkin committed:
Control server-info location
This commit enhances the control over the server-info location. It
includes the ability to enable/disable it, as well as controll the
allow list.

hunner commented on manifests/mod/info.pp:
```diff
@@ -1,9 +1,14 @@
-class apache::mod::info {
+class apache::mod::info (
+  $enable     = false,
```
Why does this disable `info.conf` by default? Not including `apache::mod::info` would also disable it so this parameter is extraneous, I think.

trlinkin replied:
The default install of apache has it disabled, at least on EL systems. I figured I'd preserve this behavior. If it makes more sense to be entirely disabled, I have no problem going that way.

trlinkin committed:
Remove enable parameter from apache::mod::info

Originally, apache::mod::info was designed with the ability to be
included but disabled. This was a misinterpretation of default apache
behavior. Ultimately, it is not required to have an enabled option.

This patch removes the 'enable' parameter from apache::mod::info. It will
now be dependent on the declaration of the class as to wether or not
mod_info is active.
